pkg/server/structlogging/hot_ranges_log_job_test.go

```diff
@@ -27,6 +27,7 @@ func TestHotRangesLoggingJobExitProcedure(t *testing.T) {
 	defer log.Scope(t).Close(t)
 	skip.UnderStress(t)
 	skip.UnderRace(t)
+	skip.IgnoreLint(t, "skipping exit test")
```

Comment:
Is there a specific reason you use `IgnoreLint` and not `skip.WithIssue(t, 149914)`?